updater.go

```diff
@@ -89,9 +90,12 @@ func (u *Updater) update(ctx Context, options UpdateOptions) (*Update, *Error) {
 		return update, promptErrPtr(err)
 	}
 	switch updateAction {
-	case UpdateActionApply, UpdateActionAuto:
-		// Continue
+	case UpdateActionApply:
+		ctx.ReportAction(UpdateActionApply, options)
+	case UpdateActionAuto:
+		ctx.ReportAction(UpdateActionAuto, options)
```

Comment:
auto is if the update was applied automatically without user prompt...

so the first time the action will be apply, and if they checked automatic, then next time it would be auto